app/scripts/modules/google/src/healthCheck/healthCheckUtils.ts

```diff
@@ -0,0 +1,44 @@
+import { IGceHealthCheck, IGceHealthCheckKind } from 'google/domain';
+
+export interface IGceHealthCheckOption {
+  displayName: string;
+  kind: IGceHealthCheckKind;
+  name: string;
+  selfLink: string;
+}
+
+export function parseHealthCheckUrl(url: string): { healthCheckName: string; healthCheckKind: IGceHealthCheckKind } {
+  const healthCheckPathParts = url.split('/');
```

Comment:
Might want to validate the length of `healthCheckPathParts`.

Reply:
:+1: Updated!